src/main/java/com/capitalone/dashboard/controller/GitHubController.java

```diff
@@ -60,4 +60,20 @@ public GitHubController(BaseCollectorRepository<Collector> collectorRepository,
                         + " Records updated: " + collector.getLastExecutionRecordCount());
     }
 
+
+    @RequestMapping(value = "/cleanup", method = GET, produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> cleanup() throws HygieiaException {
+        Collector collector = collectorRepository.findByName(GITHUB_COLLECTOR_NAME);
+        if (Objects.isNull(collector))
+            return ResponseEntity.status(HttpStatus.OK).body(GITHUB_COLLECTOR_NAME + " collector is not found");
```

Comment:
not a good practise to have logic in the controller, we would need to create a service layer and implement the DAO logic at the service layer.

Reply:
completed as suggested